packages/shorebird_tests/test/shorebird_tests.dart

```diff
@@ -177,6 +177,21 @@ $flavors
     }
   }
 
+  Future<void> runFlutterBuildIos({
+    Map<String, String>? environment,
+  }) async {
+    final result = await _runFlutterCommand(
+      // We are using `--no-codesign` because we are not signing the app
```

Comment:
This comment doesn't tell us much that we couldn't glean from the arg itself. It should probably answer questions like "why is the argument necessary" (i.e., why aren't we signing the app)

Reply:
Fair enough,  looking at the comment now, I feel dumb for writing it haha

Reply:
yeah totally, and I agree, I hate comments that say what is clear to what is being done just by the method name hahaha! Thanks for calling it out!